Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM danlynn/ember-cli:latest as build
 
-COPY package.json yarn.lock /myapp/
+COPY package.json pnpm-lock.yaml /myapp/
 
-RUN yarn install
+RUN npm install -g pnpm
+RUN pnpm install
```

Comment:
Use `--frozen-lockfile` to make sure there are no unintended changes in dependencies in the docker image.

Also, let's add `--prod` since we don't use the image for testing. This way we don't install the dev dependencies.